.travis.yml

```diff
@@ -56,6 +56,16 @@ script:
       ./scripts/build/docker/run-command.sh -r ${TARGET_ARCH} -s ${PWD} -c "xvfb-run --server-args=$XVFB_ARGS npm test && ./scripts/ci/ensure-staged-sass.sh";
     fi
 
+deploy:
+  provider: script
+  script: if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+            ./scripts/build/docker/run-command.sh -r ${TARGET_ARCH} -s ${PWD} -c "make publish-aws-s3";
+          else
+            make publish-aws-s3;
+          fi
+  on:
+    branch: master
```

Comment:
Hmmm, if somebody attempted to change this setting within a PR, is Travis actually clever enough to not 'act' on that setting until it got merged into the master-version? (and same question for Appveyor too)